R/zzz.R

```diff
@@ -0,0 +1,30 @@
+
+## nocov start
+# nolint start
+
+.onAttach <- function(libname, pkgname) {
+    packageStartupMessage(cli_inform(enw_startup_message()))
+}
+# nolint end
+
+enw_startup_message <- function() {
+    cache_location <- Sys.getenv("enw_cache_location")
+    if (check_environment_setting(cache_location)) {
+        msg <- c(
+            "!" = "`enw_cache_location` is not set. Set it using `enw_set_cache`
+            to reduce future Stan compilation times.",
+            "i" = "For example: `enw_set_cache(tools::R_user_dir(package =
```

Comment:
@medewitt agree this looks spurious. We can use # nolint (probably line wrapping here) as discussed here: https://lintr.r-lib.org/reference/exclude.html

Reply:
(also I haven't run this yet but I am excited by how cool this info message looks like it is going to be!)